CodeGeneration/Sources/SyntaxSupport/ExprNodes.swift

```diff
@@ -1683,37 +1683,50 @@ public let EXPR_NODES: [Node] = [
 
   // switch-expr -> identifier? ':'? 'switch' expr '{'
   //   switch-case-list '}' ';'?
-  //
-  // This node represents both a 'switch' expression, as well as a 'switch'
-  // statement when wrapped in an ExpressionStmt node.
   Node(
     kind: .switchExpr,
     base: .expr,
     nameForDiagnostics: "'switch' statement",
+    documentation: """
+      A `switch` statement expression.
+
+      This node represents both a 'switch' expression, as well as a 'switch'
+      statement when wrapped in an ``ExpressionStmtSyntax`` node.
```

Comment:
I would just talk about switch expressions here. Technically, there are no switch statements in SwiftSyntax.